rust/oak/src/lib.rs

```diff
@@ -124,11 +123,40 @@ mod wasm {
     // See https://rustwasm.github.io/book/reference/js-ffi.html
     #[link(wasm_import_module = "oak")]
     extern "C" {
+        pub fn wait_on_channels(buf: *const u8, count: u32) -> i32;
```

Comment:
Uhmm, IIUC `buf` is not really const, you expect to modify the received bit(s) at the very least, right?

Reply:
Oops, fixed.